docs/source/usage.rst

```diff
@@ -63,8 +63,8 @@ script to the menu, then it will appear immediately, but you will not be able to
 
 If you assign it to a toolbar icon, then it will only appear on the next start of Notepad++.
 
-Context-Sensitive help is available - if your cursor is on and notepad, editor or console function, the ``Context-Help`` menu
-will take you straight to the documentation for that function.  If the CHM file is not in the relevant location, it will open the 
+Context-Sensitive help is available - if your cursor is on a notepad, editor or console function, the ``Context-Help`` menu
+will take you straight to the documentation for that function.  If the CHM (context help manual) file is not in the relevant location, it will open the 

```

Comment:
https://en.wikipedia.org/wiki/Microsoft_Compiled_HTML_Help. In the meantime chm changed to just html help files.